zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -40,10 +40,12 @@ def __call__(
         dst: VolumetricBasedLayerProtocol,
         idx: VolumetricIndex,
     ) -> None:
-        with semaphore("read"):
-            data = src[idx]
-        with semaphore("write"):
-            dst[idx] = data
+        if src is not None:
```

Comment:
I think neither src nor dst can be None here. This op expect both src and dst, otherwise it's a user error

Reply:
OK, thanks for the review!